common/reviews/api/gulp-core-build.api.md

```diff
@@ -131,6 +131,7 @@ export interface IBuildConfig {
     libES6Folder?: string;
     libESNextFolder?: string;
     libFolder: string;
+    maxBuildTime: number;
```

Comment:
nit: `maxBuildTimeMs` since data type is number instead of timespan.

Reply:
TimeSpan isn't a type in JavaScript.